app/frontend/src/components/UI/project-list/ProjectList.tsx

```diff
@@ -23,24 +24,35 @@ const ProjectList: React.FC<Props> = ({ workspaceId, children }) => {
   const projects = useStore($projects);
 
   useEffect(() => {
-    getProjectsEffectFx({
+    getProjectsFx({
       workspaceId: workspaceId,
     });
   }, [ workspaceId ]);
 
+  /**
+   * Updates selected project data in store
+   *
+   * @param project - selected project data
+   */
+  const handleProjectClick = (project: Project | null): void => {
+    projectSelected(project);
+  };
```

Comment:
If you use currying here, it would be more readable in JSX
```suggestion
  const handleProjectClick = (project: Project | null): void => (): void =>  {
    projectSelected(project);
  };
```